orchestrator/careplanservice/handle_createtask.go

```diff
@@ -22,6 +22,14 @@ func (s *Service) handleCreateTask(httpResponse http.ResponseWriter, httpRequest
 	if err := s.readRequest(httpRequest, &task); err != nil {
 		return fmt.Errorf("invalid Task: %w", err)
 	}
+	switch task["status"] {
+	case fhir.TaskStatusDraft.String():
+		task["status"] = fhir.TaskStatusRequested.String()
```

Comment:
@bramwesselo is this a good default or should it rather be `ready`?

Reply:
Status = draft should never occur (should raise an error) because it's not an allowed state in the SCPTask profile: https://santeonnl.github.io/shared-care-planning/ValueSet-scp-task-status.html

Reply:
Fixed